apps/src/lab2/views/ExtraLinks.tsx

```diff
@@ -0,0 +1,72 @@
+import Button from '@cdo/apps/templates/Button';
+import {useFetch} from '@cdo/apps/util/useFetch';
+import React, {useEffect, useState} from 'react';
+import moduleStyles from './extra-links.module.scss';
+import ExtraLinksModal from './ExtraLinksModal';
+
+interface ExtraLinksProps {
+  levelId: number;
+}
+
+interface PermissionResponse {
+  permissions: string[];
+}
+
+interface ExtraLinksResponse {
+  links: {[key: string]: {text: string; url: string; access_key?: string}[]};
+  can_clone: boolean;
+  can_delete: boolean;
+  level_name: string;
+}
+
+// Fetch extra links for the level, then display a modal with the link data.
+const ExtraLinks: React.FunctionComponent<ExtraLinksProps> = ({
+  levelId,
+}: ExtraLinksProps) => {
+  const {loading, data} = useFetch('/api/v1/users/current/permissions');
```

Comment:
I had us get all permissions here because we eventually want to extend this to more than just levelbuilders, and it would be useful to know what permissions a user has here.